sql/updates/world/master/2022_xx_xx_xx_world.sql

```diff
@@ -17,3 +17,11 @@ DELETE FROM `creature_summoned_data` WHERE `CreatureID` IN (182084, 182088);
 INSERT INTO `creature_summoned_data` (`CreatureID`, `CreatureIDVisibleToSummoner`, `GroundMountDisplayID`, `FlyingMountDisplayID`) VALUES 
 (182084, 182085, 103104, 103104),
 (182088, 182089, 103104, 103104);
+
+-- Creature Addon
+DELETE FROM `creature_template_addon` WHERE `entry` IN (90382, 91911, 182084, 182088);
+INSERT INTO `creature_template_addon` (`entry`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `aiAnimKit`, `movementAnimKit`, `meleeAnimKit`, `visibilityDistanceType`, `auras`) VALUES
+(90382, 0, 0, 0, 1, 0, 0, 0, 0, 0, '147066'),
+(91911, 0, 0, 0, 1, 0, 0, 0, 0, 0, '147066'),
+(182084, 0, 0, 0, 1, 0, 0, 0, 0, 0, '147066'),
+(182088, 0, 0, 0, 1, 0, 0, 0, 0, 0, '147066');
```

Comment:
missing newline.

Reply:
it was added on [wline at end of SQL](https://github.com/TrinityCore/TrinityCore/pull/28085/commits/d47c3906bd6ba55b77253338d099468147dc66c7)